config.sample.yml

```diff
@@ -117,7 +117,7 @@ encryption:
   # (Optional) Configuration for encryption support in the bridge.
```

Comment:
Should the option be named to `experimental_encryption` here, or are we leaving it as-is to add even more friction to enabling it?

Reply:
And nitpick, all other config options use camelCase